src/lib/util/options.cpp

```diff
@@ -353,6 +373,15 @@ bool core_options::parse_command_line(std::vector<std::string> &args, int priori
 		bool is_unadorned = (curarg[0] != '-');
 		const char *optionname = is_unadorned ? core_options::unadorned(unadorned_index++) : &curarg[1];
 
+		// special case - collect unadorned arguments after commands into a special place
+		if (is_unadorned && !m_command.empty())
+		{
+			trim_spaces_and_quotes(args[arg]);
```

Comment:
You can't do this - the shell is responsible for trimming arguments.  If you do that here, the it will be impossible to, for example, romident a file called `"galaga" dump.zip`